lib/goodcity/user_utils.rb

```diff
@@ -0,0 +1,141 @@
+module Goodcity
+  class UserUtils
+    def self.merge_user!(master_user_id, other_user_id)
+      master_user = User.find_by(id: master_user_id)
+      other_user = User.find_by(id: other_user_id)
+
+      if master_user && other_user
```

Comment:
I would wrap this whole part in a transaction. We either want it to work completely, or not at all. We don't want it to fail halfway through and not rollback.

```
User.transaction do 
  <code to reassign user>
end
```